doozer

```diff
@@ -410,15 +413,15 @@ def images_foreach(runtime, cmd, message, push):
 
     \b
     The following environment variables will be available in each invocation:
-    oit_repo_name : The name of the distgit repository
-    oit_repo_namespace : The distgit repository namespaces (e.g. containers, rpms))
-    oit_config_filename : The config yaml (basename, no path) associated with an image
-    oit_distgit_key : The name of the distgit_key used with -i, -x for this image
-    oit_image_name : The name of the image from Dockerfile
-    oit_image_version : The current version found in the Dockerfile
-    oit_group: The group for this invocation
-    oit_metadata_dir: The directory containing the oit metadata
-    oit_working_dir: The current working directory
+    doozer_repo_name : The name of the distgit repository
+    doozer_repo_namespace : The distgit repository namespaces (e.g. containers, rpms))
+    doozer_config_filename : The config yaml (basename, no path) associated with an image
+    doozer_distgit_key : The name of the distgit_key used with -i, -x for this image
```

Comment:
Are these all actually lowercase, or uppercase?

Reply:
lowercase, those used to be `oit_*`, I just did a find/replace in that section. Honestly that whole foreach command is in dire need of an overhaul. I'm not 100% sure it works as intended anymore. It's very old.